Sources/TensorFlow/Layers/Recurrent.swift

```diff
@@ -200,6 +200,80 @@ public struct LSTMCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
     }
 }
 
+/// An GRU cell.
+public struct GRUCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
+    public var updateWeight1, updateWeight2: Tensor<Scalar>
+    public var resetWeight1, resetWeight2: Tensor<Scalar>
+    public var outputWeight1, outputWeight2: Tensor<Scalar>
+    public var updateBias, outputBias, resetBias: Tensor<Scalar>
+
+    @noDerivative public var stateShape: TensorShape {
+        [1, updateWeight.shape[0]]
+    }
+
+    public var zeroState: State {
+        State(hidden: Tensor(zeros: stateShape))
+    }
+
+    public typealias TimeStepInput = Tensor<Scalar>
+    public typealias TimeStepOutput = State
+    public typealias Input = RNNCellInput<TimeStepInput, State>
+    public typealias Output = RNNCellOutput<TimeStepOutput, State>
+
+    /// Creates a `GRUCell` with the specified input size and hidden state size.
+    ///
+    /// - Parameters:
+    ///   - inputSize: The number of features in 2-D input tensors.
+    ///   - hiddenSize: The number of features in 2-D hidden states.
+    public init(
+        inputSize: Int,
+        hiddenSize: Int,
+        seed: TensorFlowSeed = Context.local.randomSeed
+    ) {
```

Comment:
Could you switch to using the initialization convention we use for other layers? See, for example, how the [`Dense` layer](https://github.com/tensorflow/swift-apis/blob/4547df8ec3129b145bcb51b4ee6a7b9daebf8021/Sources/TensorFlow/Layers/Core.swift#L131) initializers are defined.

Reply:
Could you be more specific? I see some small differences between the initializers, but I'm not exactly sure which conventions you are referring to.

Reply:
Sorry for the confusion. I was referring to the initialization method being used (e.g., `zeros` vs `glorotUniform` vs others). The approach followed for other layers allows the user to provide a custom initialization method for the layer parameters if they want to. You should also modify this one to support custom initialization methods.

Reply:
@eaplatanios Sorry, I am still confused. Could you tell me if I am understanding you correctly?

In other words, you would like me to add the weights and biases to the initializer so that someone could initialize them differently?

Reply:
@eaplatanios @sgugger @marcrasi This is the last request on this PR, could you give me some requested info so that we can get this merged in. Thanks 😄

Reply:
Sorry I missed that comment. The requested change is to pass a `weightInitializer` and `biasInitializer` in this init, on top of the sizes and use them (no need to take a `seed` then, since you can seed your initiliazer). Here is an example for the initialization of [`Dense`](https://github.com/tensorflow/swift-apis/blob/4547df8ec3129b145bcb51b4ee6a7b9daebf8021/Sources/TensorFlow/Layers/Core.swift#L171):
``` 
init(
        inputSize: Int,
        outputSize: Int,
        activation: @escaping Activation = identity,
        weightInitializer: ParameterInitializer<Scalar> = glorotUniform(),
        biasInitializer: ParameterInitializer<Scalar> = zeros()
    ) {
        self.init(
            weight: weightInitializer([inputSize, outputSize]),
            bias: biasInitializer([outputSize]),
            activation: activation)
    }
```

Reply:
@sgugger Thanks for the help! Everything should be finished now.